### PR TITLE
chore: automate generis extension release

### PR DESCRIPTION
**Pull request by gabrielfs7**
# Goal 

Allow generis extension to be released automatically

# Why?

- We are spending to much time on releasing it manually
- There is no much benefit on it as many other core extensions also have automatic release

### CODE REVIEW

**Review by wazelin — CHANGES_REQUESTED** [review]
While we're at it, let's also add a workflow similar to https://github.com/oat-sa/tao-core/blob/master/.github/workflows/conventional_commits_check.yml. Having a release version preview in the PR might come in handy.

File: .github/workflows/release_tao_extension.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if: github.event.pull_request.merged == true
+    name: Automated Tao extension release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Release
+        uses: oat-sa/extension-release-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```
Comment (wazelin) [inline .github/workflows/release_tao_extension.yml:21]: That's no-longer required is it?

Comment (gabrielfs7) [reply .github/workflows/release_tao_extension.yml:21]: I am not sure, but tao-core and other extensions use it, so I am not sure we should remove.

Comment (wazelin) [reply .github/workflows/release_tao_extension.yml:21]: Fine by me. I see no evidence of that env var being used, but sure, it won't hurt keeping it.

**Review by wazelin — APPROVED** [review]

**Pull request merged**